client/components/Footer.js

```diff
@@ -30,7 +30,7 @@ const Footer = () => {
                     </ChakraLink>
                 </Flex>
             </Flex>
-            <Flex flexDirection="row" justifyContent="space-between" alignItems="center" paddingMargin={1}>
+            <Flex flexDirection="row" justifyContent="space-between" alignItems="center" paddingmargin={1}>
```

Comment:
This was throwing a warning message in the console so I just edited it. Will take a closer look next time if I have time